packages/jest-environment-puppeteer/src/PuppeteerEnvironment.js

```diff
@@ -47,7 +47,21 @@ class PuppeteerEnvironment extends NodeEnvironment {
           : undefined,
       browserWSEndpoint: wsEndpoint,
     })
-    this.global.page = await this.global.browser.newPage()
+
+    if(config.browserContext === 'incognito') {
+      // Using this, pages will be created in a pristine context.
+      this.global.context = await this.global.browser.createIncognitoBrowserContext()
+    } else if(config.browserContext === 'default') {
+      /**
+       * Since this is a new browser, browserContexts() will return only one instance
+       * https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#browserbrowsercontexts
+       */
+      this.global.context = await this.global.browser.browserContexts()[0]
+    } else {
+      throw new Error(`browserContext should be either 'incognito' or 'default'. Received '${config.browserContext}'`)
```

Comment:
It is actually a breaking change because there is no default value specified and all users will have an error.

Please consider it as `default` if not specified.

Reply:
Will fix soon ! Thanks for the feedback :) !